samples/choreo-token-exchange-react-app/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```

Comment:
Shall we update the README file instructions relevant to the sample app and a guide on how to configure the server?